profile_bluesky/startup/instrument/devices/lakeshore340.py

```diff
@@ -0,0 +1,152 @@
+"""
+Lakeshore temperature controllers
+"""
+
+from instrument.session_logs import logger
+logger.info(__file__)
+
+from apstools.synApps.asyn import AsynRecord
+from ophyd import Component, Device, Signal
+from ophyd import EpicsSignal, EpicsSignalRO
+from ophyd import FormattedComponent,PVPositioner
+from ophyd import Kind
+
+class DoneSignal(Signal):
```

Comment:
Defined twice: seems like a common utility signal.  Does this need to be a Signal?

Reply:
Good point on being defined twice, I will move to the _utils_ folder.

I'm not sure that it has to be a Signal. It does need to be subscribed to the `PVPositioner.readback` and `PVPositioner.setpoint`, and it needs to have `PVPositioner._move_changed` subscribed to it.